packages/mc-scripts/src/cli.ts

```diff
@@ -112,7 +112,7 @@ async function run() {
       'Compiles "index.html.template" file into a "index.html" with all the required runtime configuration. The security headers are also compiled and injected into the "index.html".'
     )
     .option(
-      '--transformer [path]',
+      '--transformer <path>',
```

Comment:
Minor change to being more strict as these option arguments are supposed to be required.